src/EditorFeatures/CSharpTest/InitializeParameter/AddParameterCheckTests.cs

```diff
@@ -275,8 +276,11 @@ class C
             await VerifyCS.VerifyRefactoringAsync(code, code);
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInitializeParameter)]
-        public async Task TestNotOnPartialMethodDefinition1()
+        [Theory]
+        [Trait(Traits.Feature, Traits.Features.CodeActionsInitializeParameter)]
+        [InlineData(LanguageVersionExtensions.CSharpNext)]
+        [InlineData(LanguageVersion.CSharp8)]
```

Comment:
```suggestion
        [CombinatorialData]
```

Reply:
This ends up giving us unexpected diagnostics about use of partial methods in C# 2.